src/js/common/sentry.js

```diff
@@ -0,0 +1,22 @@
+import Raven from 'raven-js';
+
+import environment from './helpers/environment';
+
+// url check is necessary for e2e tests
+const trackErrors = (__BUILDTYPE__ === 'production' || __BUILDTYPE__ === 'staging')
```

Comment:
I think this is the logic we'd want, but need some guidance. My goal was to have this enabled only in staging and production, and not on dev, locally, or in tests.